### PR TITLE
added Footer component, components & styles folders

### PR DESCRIPTION
**Pull request by davetyson**
- made a components folder
- added Footer to components folder
- made a styles folder
- added footer.css to styles folder but left it blank for now
- added a Footer component to the return statement of the App.js to put it on the page

### CODE REVIEW

**Review by tywy87 — COMMENTED** [review]
Looks good to me Dave!

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    // useEffect to check for the current year on page load and generate that
+    useEffect( () => {
+        setCurrentYear(new Date().getFullYear());
+    }, []);
+
+    return (
+        // Footer generated with the current year from the useEffect above
+        <footer>
+            <p>© 3 Beards, 1 Moustache {currentYear}. Created at Juno College.</p>
```
Comment (allan-viveiros) [inline src/components/Footer.js:21]: Like the group name on footer! :)

**Review by allan-viveiros — APPROVED** [review]
That's all good!

**Comment by stevesidoli** [comment]
Approved

**Pull request merged**